infrastructure/running-a-validator/operational-tasks/general-management.md

```diff
@@ -319,15 +319,11 @@ The optional `Alertmanager` complements Prometheus by handling alerts and notify
     sudo mkdir /etc/alertmanager
     sudo nano /etc/alertmanager/alertmanager.yml
     ```
-    Add the following code to the configuration file to define email notifications:
+    You must generate an [`app password` in your Gmail account](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank} to allow `Alertmanager` to send you alert notification emails. Once you have the `app password`, add the following code to the configuration file to define email notifications:
```

Comment:
```suggestion
    You must generate an [app password in your Google account](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank} to allow `Alertmanager` to send you alert notification emails. Once you have the app password, add the following code to the configuration file to define email notifications:
```

I don't really think we should be putting Alertmanager in back ticks, but that's a problem for another day. I'll take care of this in a separate PR

Reply:
FYI, I created a ticket for this and started to work on it in a separate branch